controller/node_controller.go

```diff
@@ -387,13 +405,20 @@ func (nc *NodeController) syncDiskStatus(node *longhorn.Node) error {
 			return err
 		}
 		if !nc.scheduler.IsSchedulableToDisk(0, info) {
-			logrus.Errorf("unable to schedule any replica on node %v", node.Name)
+			logrus.Errorf("unable to schedule any replica to disk %v on node %v", disk.Path, node.Name)
 			if condition.Status != types.ConditionStatusFalse {
 				condition.LastTransitionTime = util.Now()
 			}
 			condition.Status = types.ConditionStatusFalse
 			condition.Reason = string(types.DiskConditionReasonDiskPressure)
 			condition.Message = fmt.Sprintf("the disk %v on the node %v has %v available, but requires reserved %v, minimal %v to schedule more replicas", disk.Path, node.Name, diskInfo.StorageAvailable, disk.StorageReserved, minimalAvailablePercentage)
+		} else if readyCondition.Status != types.ConditionStatusTrue {
```

Comment:
It should be automatic since we've set `StorageMaximum` to 0.

Reply:
If all values are 0, it could schedule a 0 size replica. So I add ready check for scheduler.